src/components/videoScreen/VideoScreen.tsx

```diff
@@ -0,0 +1,130 @@
+import s from "./videoScreen.module.scss";
+import MicOff from "../../assets/MicOff.svg";
```

Comment:
For readability of the code, we follow the imports, first absolute, then local, individual entities can be highlighted with a comment

Reply:
fixed